src/coreclr/scripts/superpmi.py

```diff
@@ -1492,6 +1493,7 @@ def replay_with_asm_diffs(self):
                 with ChangeDir(self.coreclr_args.core_root):
                     command = [self.superpmi_path] + flags + [self.base_jit_path, self.diff_jit_path, mch_file]
                     return_code = run_and_log(command)
+                    logging.info("return_code: %s", return_code)
```

Comment:
Either remove this, or change it to `logging.debug`

Reply:
Changed to debug.